### ISSUE TITLE
A new feature

### ISSUE THREAD

**Issue by Gorfar** [vdrhtc/concrypt#4]
I consider that statistics should be implemented at the top of the main screen., e.g. number of friends online, the total number of messages, some ratio of your produtivity (sent/received) or smth else. It's possible that it would be interesting for Vk consumers.

**Comment by vdrhtc** [comment]
Yes, I also thought about that, also about some graphs of a friends' online status for example to see who is the leader of time wasting))

**Comment by Gorfar** [comment]
To get to know who is "the very weakly"

**Comment by vdrhtc** [comment]
))))

**Comment by Gorfar** [comment]
And also a number of new friends.
